### ISSUE TITLE
Documentation please!

### ISSUE THREAD

**Issue by seaniedan** [xychelsea/deepfacelab-docker#13]
Perhaps I'm misunderstanding something fundemental here about how to use these shell scripts - could you please write some very brief instructions on what do do after running the Docker image to run the scripts?

**Comment by seaniedan** [comment]
Another insight: if I enter the docker container using
docker exec -i -t deepfacelab /bin/bash
I can run the scripts:
cd scripts && ./4.2_data_src_sort.sh

...I just can't use the Jupyter notebook as a tool, as if I do:
import numpy or
import colorama
I get 'module not found' errors.

**Comment by xychelsea** [comment]
Yeah, I need to be better about this. Sorry I only have time to periodically come back to this project every few months. I am going to fix these issues this week. Thank you.

**Comment by xychelsea** [comment]
I neglected to set up the right base container and neglected to install kernel management in jupyter notebooks, version v0.4.1 will correct this
